src/test/java/browsertests/day1.java

```diff
@@ -0,0 +1,16 @@
+package browsertests;
+
+import org.testng.annotations.Test;
+
+public class day1 {
```

Comment:
Classes should start with capital letters and not include numbers.

Consider naming this to `DayOne`